Tests/test_SeqIO_features.py

```diff
@@ -735,8 +732,7 @@ def test_fuzzy_join(self):
 
         f1 = SeqFeature(FeatureLocation(AfterPosition(310), 320), strand=-1)
         # Note - is one-of(340,337) allowed or should it be one-of(337,340)?
-        f2 = SeqFeature(FeatureLocation(325, OneOfPosition(340, [ExactPosition(340),
-                                                                ExactPosition(337)])),
+        f2 = SeqFeature(FeatureLocation(325, OneOfPosition(340, [ExactPosition(340), ExactPosition(337)])),
```

Comment:
Ideally as per PEP8 https://www.python.org/dev/peps/pep-0008/#maximum-line-length aim for 79 characters, but we and PEP8 allow flexibility here as needed to prioritised readability.

Note that our TravisCI/Tox setup currently runs ``pep8 --max-length ...`` explicitly on some parts of the code, but ``Bio/`` in particular as a whole still has some extremely long lines so we can't enforce this globally.